devel/glib2-devel/Portfile

```diff
@@ -80,9 +81,10 @@ configure.python            ${prefix}/bin/python3.7
 configure.env-append        PERL_PATH=${configure.perl}
 
 post-patch {
+    reinplace -W ${worksrcpath} "s|@PYTHON@|${configure.python}|" meson.build
     reinplace -W ${worksrcpath} "s|@PREFIX@|${prefix}|g" gio/gdbusprivate.c gio/xdgmime/xdgmime.c glib/gi18n-lib.h glib/gi18n.h gio/gdbusaddress.c
     reinplace "s|data_dirs = \"/usr|data_dirs = \"${prefix}/share:/usr|g" ${worksrcpath}/glib/gutils.c
-    reinplace "s|path = \"/bin|path = \"${prefix}/bin:/bin|g" ${worksrcpath}/glib/gutils.c ${worksrcpath}/glib/gspawn.c
+    reinplace "s|path = \"/bin|path = \"${prefix}/bin:/bin|g" ${worksrcpath}/glib/gutils.c ${worksrcpath}/glib/gspawn.c    
```

Comment:
There is a whitespace issue at the end of this line

Reply:
Fixed.